components/zigbee/time/__init__.py

```diff
@@ -0,0 +1,27 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import time as time_
+from esphome.cpp_generator import get_variable
+from esphome.const import (
+    CONF_ID,
+    CONF_DEVICE
+)
+
+DEPENDENCIES = ["zigbee"]
+
+zigbee_ns = cg.esphome_ns.namespace("zigbee")
+ZigbeeTime = zigbee_ns.class_("ZigbeeTime", time_.RealTimeClock)
+ZigBeeComponent = zigbee_ns.class_("ZigBeeComponent", cg.Component)
+CONFIG_SCHEMA = time_.TIME_SCHEMA.extend(
+    {
+        cv.GenerateID(): cv.declare_id(ZigbeeTime),
+        cv.Required(CONF_DEVICE): cv.use_id(ZigBeeComponent),
```

Comment:
```suggestion
        cv.GenerateID(CONF_ZIGBEE_ID): cv.use_id(ZigBeeComponent),
```
Add `CONF_ZIGBEE_ID = "zigbee_id"` to the zigbee init file and import it here.